layouts/index.html

```diff
@@ -9,8 +9,7 @@
         {{ end }}
 
         <div class="posts-list">
-          {{ $pag := .Paginate (where .Data.Pages "Type" "post") }}
-          {{ range $pag.Pages }}
+          {{ range where .Paginator.Pages "Type" "post" }}
```

Comment:
It's better to use `site.Params.mainSections` instead of hard-coded `"post"`.

Reply:
I updated the code